### ISSUE TITLE
Consider `alert` as option instead of audio cue for components

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#195282]
@jooyoungseo and @rperez030 brought to my attention the fact that audio cues might not be practical for certain users. 

We should consider adding `alert` when audio cues are disabled for various features.

**Comment by rperez030** [comment]
I agree. Breakpoints, errors and warnings come to mind. I would also add that audio cues and screen reader alert probably shouldn't be seen as mutually exclusive. For example, one thing I love about the diff view experience is the multimodal feedback. For example, when I move cursor to an inserted line, I get the ascending tone, the plus sign in my Braille display and the screen reader telling me  "modified line", all of that at once. I have no way to get it wrong. Even with audio cues enabled, I've never managed to tell the difference between errors and warnings. If I hear the 2 audio cues together i can tell the difference, but don't ask me five minutes later. Is that a me thing? Maybe something to investigate.

**Comment by meganrogge** [comment]
@rperez030 and @jooyoungseo, should we have the alerts for breakpoint on line, error on line, and warning on line on by default?

**Comment by jooyoungseo** [comment]
I think we can turn on the alert by default, allowing users to turn off if needed.

**Comment by rperez030** [comment]
A resounding yes from me!!! I need to catch up with this thread, but I wonder if "on line" is really necessary. to me "error, warning and breakpoint followed by an empty space should be enough. This is something we should test very carefully in all the platforms with all screen readers before it hits production.Super happy to help with that in January.

**Comment by meganrogge** [comment]
@jooyoungseo and @rperez030 do you think this is something users will want to configure per audio cue/ alert? I worry that's a lot of settings to have to manage for the user. Perhaps there should be one overarching setting with values of `audioCue`, `alert`, or `none`, or `both`. Thoughts?

**Comment by meganrogge** [comment]
Edit, the setting would be something like `alert`: `on`, `off` similar to what we do for audio cues, only this would apply across the board.

**Comment by meganrogge** [comment]
Note that we will not have such settings for inline suggestion, chat response pending, or chat response received as the inline suggestion should always be alerted, response pending is not needed IMO, and response received is indicated via an alert of the response.

**Comment by meganrogge** [comment]
this was reverted for re-engineering
